### ISSUE TITLE
Character pseudonyms

### ISSUE THREAD

**Issue by Throne3d** [Marri/glowfic#49]
Examples: Glam/Sadde, Theo?/Thea?

Also allow names to be concealed for certain replies (requested [here](http://alicorn.elcenia.com/board/viewtopic.php?f=12&t=459&p=28533#p28532)).

>I think a more general "let the lines displayed to identify a character be customisable" is ideal? At least the lines that currently have "name" and "screen name"

-Pedro

**Comment by Throne3d** [comment]
The pseudonym should default to whatever pseudonym that character last used in that thread. It would be useful to have a dropdown of recent pseudonyms / all pseudonyms so far used by the particular character in the thread, so you can switch back and forth easily, as necessary, but this would require more UI.

Perhaps it should go in the character picker dropdown, and it should maybe also use the same mechanism for NPC posting (using the user account, but with a customizable 'character name' bar that doesn't link anywhere?).

**Comment by Marri** [comment]
Making a new issue for the minor improvements remaining after #106 got merged
